e2e/__tests__/crawlSymlinks.test.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {tmpdir} from 'os';
+import * as path from 'path';
+import * as fs from 'fs';
+import {writeFiles, writeSymlinks} from '../Utils';
+import runJest from '../runJest';
+const DIR = path.resolve(tmpdir(), 'crawl-symlinks-test');
+
+test('Node crawler picks up symlinked files', () => {
+  if (fs.existsSync(DIR)) {
```

Comment:
Should have a `beforeAll` setting it up and an `afterAll` cleaning it up. Can use rimraf for removing, I think that's what we use elsewhere

Reply:
Done!